test/pylib/rest_client.py

```diff
@@ -162,6 +162,17 @@ async def get_host_id_map(self, dst_server_ip: IPAddress) -> list[HostID]:
         assert(type(data) == list)
         return data
 
+    async def get_ownership(self, dst_server_ip: IPAddress, keyspace: str = None, table: str = None) -> list:
+        """Retrieve the ownership"""
+        if keyspace is None and table is None:
+            api_path = f"/storage_service/ownership/"
+        elif table is None:
+            api_path = f"/storage_service/ownership/{keyspace}"
+        else:
+            api_path = f"/storage_service/ownership/{keyspace}?cf={table}"
```

Comment:
This breaks for keyspace is None and table is not None.

Also, it's excessive splitting, adding a simple unused function.